spec/features/reviews_spec.rb

```diff
@@ -0,0 +1,124 @@
+# require "rails_helper"
+# require "pry"
+# ​
+#   feature "Create A Review" do
+#     scenario "logged in user successfully creates a new review" do
+#
+#       new_user = User.create(email: "mckelvey.matt@gmail.com", password: "12345678")
+#     ​
+#       visit user_session_path
+#     ​
+#       fill_in 'Email', with: 'mckelvey.matt@gmail.com'
+#       fill_in 'Password', with: '12345678'
+#     ​
+#       click_button "Log in"
+#     ​
+#       click_button "Add New Bar"
+#     ​
+#       fill_in 'Name', with: 'Beantown Pub'
+#       fill_in 'Description', with: 'After hours cocktail bar with pool.'
+#     ​
```

Comment:
The tests are commented out as the reviews have not been linked.  Will be working in the next commit.